app/src/main/java/com/example/ubclaunchpad/launchnote/BaseActivity.kt

```diff
@@ -73,9 +75,37 @@ abstract class BaseActivity : AppCompatActivity() {
 
     abstract fun getContentViewId(): Int
 
+    override fun onActivityResult(requestCode: Int, resultCode: Int, data: Intent?) {
```

Comment:
You can make the method signature  'onActivityResult(requestCode: Int, resultCode: Int, data: Intent)' instead, so that you don't have to access data with !!.